### PR TITLE
Clean up: software version

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- establish alphabetical order

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/products/softwareVersion.schema.tpl.json [hunk 1]
```diff
@@ -122,7 +113,7 @@
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add the content types of all possible input formats for this software version.",
+      "_instruction": "Add all content types that can be generated as output for this software version.",
```
Comment (lzehl) [inline schemas/products/softwareVersion.schema.tpl.json]: "Add all content types that can be generated as output by this software version." ??

File: schemas/products/softwareVersion.schema.tpl.json [hunk 2]
```diff
@@ -53,25 +53,16 @@
...
-      "items": {
-        "type": "string"
-      }
-    },
     "inputFormat": {
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add the content types of all possible input formats for this software version.",
+      "_instruction": "Add all content types that can be used as input for this software version.",
```
Comment (lzehl) [inline schemas/products/softwareVersion.schema.tpl.json]: "Add all content types that can be used as input by this software version." ??

File: schemas/products/softwareVersion.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/products/softwareVersion.schema.tpl.json]: changed it to "by"

File: schemas/products/softwareVersion.schema.tpl.json (hunk 2 above)
Comment (lzehl) [reply schemas/products/softwareVersion.schema.tpl.json]: changed it to "by"

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison or @Peyman-N or @aeidi89 feel free to merge if you also approve

**Review by aeidi89 — APPROVED** [review]
Looks good to me.

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**